BSP/buttons/buttons.h

```diff
@@ -0,0 +1,30 @@
+#ifndef BUTTONS_H
+#define BUTTONS_H
+
+#include <stdbool.h>
+
+typedef enum button{
```

Comment:
I think this is not needed
```suggestion
typedef enum{
```